README.md

```diff
@@ -68,19 +81,27 @@ steps:
       ocaml_version: ${{ matrix.ocaml_version }}
 ```
 
+Each field can be customized, see below
+for the documentation of those specific to the docker-coq-action,
+or the GitHub Actions official documentation for the
+[standard fields involved in workflows](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions).
+
+See [action.yml](./action.yml)
+
 See also the [example repo](https://github.com/erikmd/docker-coq-github-action-demo).
 
-### Inputs
+### Inputs to the action.yaml file
+
+#### `opam_file` *Optional*
```

Comment:
I agree with this change (but just to do the remark it will change the implied "permalinks"… 😅)